coalib/bearlib/languages/definitions/html.py

```diff
@@ -0,0 +1,14 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class HTML:
+    __qualname__ = 'Hypertext Markup Language'
+    versions = 2.0, 3.2, 4.0, 4.01, 5, 5.1
+
+    extensions = '.html', '.html'
+    multiline_comment_delimiters = {'<!--': '-->'}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = string_delimiters
```

Comment:
Maybe we should merge these two lines to

```python
multiline_string_delimiters = string_delimiters = {'"': '"', "'": "'"}
```

but this shouldn't block the merge 🙃